autoload/airline/themes/substrata.vim

```diff
@@ -0,0 +1,95 @@
+" Copyright (C) 2019-present Aramis Razzaghipour <aramisnoah@gmail.com>
+
+" Project: Substrata Vim
+" Description: A cold and dark colorscheme for vim-airline-themes inspired
+" by Nord and Iceberg
+" Repository: https://github.com/arzg/vim-substrata
+" License: ISC
+
+let s:substrata_vim_version="1.0"
+let g:airline#themes#substrata#palette = {}
+
+let s:substrata0_gui = "#2e313d"
+let s:substrata1_gui = "#3c3f4e"
+let s:substrata2_gui = "#5b5f71"
+let s:substrata3_gui = "#6c6f82"
+let s:substrata4_gui = "#b5b4c9"
+let s:substrata5_gui = "#f0ecfe"
+let s:substrata6_gui = "#f5ecfe"
+let s:substrata7_gui = "#659ea2"
+let s:substrata8_gui = "#a0b9d8"
+let s:substrata9_gui = "#8296b0"
+let s:substrata10_gui = "#7dc2c8" 
+let s:substrata11_gui = "#fe9f7c"
+let s:substrata12_gui = "#cf8164"
+let s:substrata13_gui = "#d2b45f"
+let s:substrata14_gui = "#92c47e"
+let s:substrata15_gui = "#c6aed7"
```

Comment:
These colours don’t match with the palette in the README. For example, color15 in the official palette is `#f0ecfe`, while here it is `#c6aed7`, which is the value used for colour18, Bright Magenta. Is this intentional?

Reply:
I used the theme link into the issues, but I'll change this color tonight ! :)

Reply:
Since its most of the color that are not the same I'd ask [hackedwifi](https://github.com/hackedWifi) since it is the person who make it. If you prefer, we can just close this PR since it's not the same colors. :)